commons-lang/src/main/java/org/apache/commons/lang3/LocaleUtils.java

```diff
@@ -137,7 +137,7 @@ private static boolean isISO3166CountryCode(final String str) {
      * @return true, if the given String is a ISO 639 compliant language code.
      */
         private static boolean isISO639LanguageCode(final String str) {
-            return StringUtils.isAllLowerCase(str) &(str.length() == 2 || str.length() == 3);
+            return StringUtils.isAllLowerCase(str) && (str.length() == 2 || str.length() == 3);
```

Comment:
`return StringUtils.isAllLowerCase(str) & (str.length() == 2 || str.length() == 3);`

Reply:
Thank you @mruchann for your suggestion, can you please explain why my suggestion is wrong? I think a singular ampersand is a bitwise AND and a pair of ampersand is the logical AND.
I also looked into the other methods in the source file whose return type was boolean, and those methods used `&&`. From that context I put double ampersand

Reply:
Thanks for pointing it out @mruchann